k8s/jiva/README.md

```diff
@@ -0,0 +1,124 @@
+This document will help to delete the auto-generated snapshots created when a Jiva replica restart happened or new replica got added to the Jiva controller. The steps for deleting the auto-generated snapshots  are the following:
+
+Get the details of Jiva controller pod using the following command.
+
+```
+kubectl get pod -n <PVC_namespace>
+```
+Example: 
+```
+kubectl get pod
```

Comment:
To keep it consistent with above syntax, say `kubectl get pod -n default` or show an example that is running in an example namespace.

Reply:
Done. Please check.